### PR TITLE
Sanita r

### PR DESCRIPTION
**Pull request by Roccubalbetta**

### CODE REVIEW

**Comment by eutampieri** [comment]
The problems I’ve raised before seems fixed, no style checking was done though

File: Riccardo_Sanita/sanita/Program.cs [hunk 1]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Tampieri
```
Comment (eutampieri) [inline Riccardo_Sanita/sanita/Program.cs]: No 🙅‍♂️

**Review by eutampieri — CHANGES_REQUESTED** [review]

File: Riccardo_Sanita/sanita/Program.cs (hunk 1 above)
Comment (Roccubalbetta) [reply Riccardo_Sanita/sanita/Program.cs]: ahahahah sorry

Comment (Roccubalbetta) [reply Riccardo_Sanita/sanita/Program.cs]: fixed

**Review by eutampieri — APPROVED** [review]

**Pull request merged**